packages/react/src/hooks/useStepHandlers.ts

```diff
@@ -37,9 +49,9 @@ export function useStepHandlers(step: FlowStep, { onPrimary, onSecondary }: Step
 
   return {
     handlePrimary: useCallback<StepHandler>(
-      async (e, properties) => {
-        const continueDefault = await onPrimary?.(step, e)
-
+      async (e, properties, optimistic) => {
```

Comment:
Can use default value `(e, properties, optimistic = true)` here instead of `const optimisticValue` below.

Reply:
Good catch.